pkg/service/v0/settings.go

```diff
@@ -30,6 +31,17 @@ func RegisterSettingsBundles(l *olog.Logger) {
 			l.Info().Str("bundle", res.Bundle.Id).Msg("Successfully registered bundle")
 		}
 	}
+
+	permissionRequests := generateProfilePermissionsRequests()
+	for i := range permissionRequests {
+		res, err := service.AddSettingToBundle(context.Background(), &permissionRequests[i])
+		bundleID := permissionRequests[i].BundleId
+		if err != nil {
+			l.Err(err).Str("bundle", bundleID).Str("setting", permissionRequests[i].Setting.Id).Msg("Error adding setting to bundle")
+		} else {
+			l.Info().Str("bundle", bundleID).Str("setting", res.Setting.Id).Msg("Successfully added setting to bundle")
```

Comment:
make that `Debug` ? otherwise the logs will get boring quickly... unless this is happening only once at the start of the service, in which case it's ok

Reply:
Only happens once on service start.